demos/demo_robot_impedance.py

```diff
@@ -12,6 +12,7 @@
 from mim_control.robot_impedance_controller import RobotImpedanceController
 from bullet_utils.env import BulletEnvWithGround
 from robot_properties_solo.solo12wrapper import Solo12Robot, Solo12Config
+from robot_properties_go1.go1wrapper import Go1Robot, Go1Config
```

Comment:
would it be possible to add a try catch here for the optional dependencies?

for example one wants to run on solo but not on go1 and still need to install go1 to run the demo...
It would be  nice to allow the users to run it without solo if you run go1 and the other way around.

WDYT?